docs/design/template.md

```diff
@@ -134,9 +134,16 @@ Some of these prompts may not be relevant to your design document; in which case
   - Can the new functionality affect clusters which are not explicitly using it?
   - What testing and safe guards are being put in place to protect against unexpected problems?
 
-- Security questions:
-  - Does the change concern authentication or authorization logic? If so, mention this explicitly tag the relevant security-minded reviewer as reviewer to the design document.
-  - Does the change create a new way to communicate data over the network?  What rules are in place to ensure that this cannot be used by a malicious user to extract confidential data?
+- Security questions (also go through [OWASP TOP 10](https://owasp.org/Top10/) and [CWE Top 25](https://cwe.mitre.org/top25/) lists):
+  - Is the functionality accessible through the client protocol? What permissions (types/names/actions) will be used to protect the access?
```

Comment:
We also have SQL - e.g. the functionality may introduce new `CREATE` keywords, which may need permissions.